css/style.css

```diff
@@ -217,3 +227,370 @@
 }
 
 /* ********************** end of desktop version ********************** */
+
+/*  ******** PAGES 2 AND 3 ****** */
+
+.works_main {
+  padding: 8.13rem 1rem;
+  background-color: var(--color-background_clear);
+}
+
+.works_title {
+  background-color: var(--color-background_clear);
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 3.25rem;
+  text-align: center;
+}
+
+.line_div {
+  display: block;
+  border-bottom: solid 2px var(--color-back-content1);
+  width: 100%;
+  margin: 4rem auto;
+}
+
+.grid_works {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(327px, 1fr));
+  grid-gap: 2rem;
+}
+
+.main_card_work {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  margin-bottom: 2.8rem;
+}
+
+.card_cnt {
+  display: flex;
+  flex-direction: column-reverse;
+  background-size: cover;
+  min-height: 400px;
+  max-height: 425px;
+}
+
+.work_img {
+  width: 100%;
+  margin-bottom: 2rem;
+}
+
+.main_card {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  order: 1;
+  margin: 0 auto;
+}
+
+.work-1 {
+  background: url("../images/img-ph1.png") no-repeat center;
+}
+
+.work-2 {
+  background: url("../images/img-ph2.png") no-repeat center;
+}
+
+.work-3 {
+  background: url("../images/img-ph3.png") no-repeat center;
+}
+
+.work-4 {
+  background: url("../images/img-ph2.png") no-repeat center;
+}
+
+.work-5 {
+  background: url("../images/img-ph3.png") no-repeat center;
+}
+
+.work-6 {
+  background: url("../images/img-ph2.png") no-repeat center;
+}
+
+/* ************** Fonts and config for cards ******************* */
+.cnt_info_card {
+  padding-left: 1rem;
+}
+
+.Title1_card {
+  color: var(--color-font-title1_card);
+  font-family: var(--font-title1);
+  font-size: 2rem;
+  font-weight: 300;
+}
+
+.Title2_card {
+  color: var(--color-font-title2_card);
+  font-family: var(--font-title2);
+  font-size: 1.3rem;
+  padding-bottom: 1rem;
+}
+
+.Simpletext_card {
+  color: var(--color-font-text1_card);
+  font-family: var(--font-text1);
+  font-style: normal;
+  font-size: 0.85rem;
+}
+
+/********************** buttons for card ********************/
+
+.project_btn_all {
+  display: block;
+  background-color: var(--color-font-title1);
+  font-family: var(--font-title2);
+  font-weight: bold;
+  padding: 0.75rem;
+  text-decoration: none;
+  color: var(--color-font-menu);
+  transition: transform 0.4s ease;
+  text-align: center;
+}
+
+.project_btn_1 {
+  font-weight: bold;
+  text-decoration: none;
+  color: var(--color-font-menu);
+  width: fit-content;
+  transition: transform 0.4s ease;
+}
+
+.project_btn_2 {
+  color: var(--color-font-menu);
+  font-weight: bold;
+  text-decoration: none;
+  transition: transform 0.4s ease;
+}
+
+.project_btn_all:hover {
+  transform: translateY(-0.5rem);
+  cursor: pointer;
+}
```

Comment:
Please consider using the same styles for `hover and press` functionality as shown in the Figma Template to make it more closely align with the professional template. 

## DESIGN HOVER(:**hover**) and PRESS(:**active**) GUIDELINES:-

![image](https://user-images.githubusercontent.com/94223367/227763427-2f5861bb-c5ce-43b1-b062-7e83168cd16c.png)